jest.config.js

```diff
@@ -13,12 +13,17 @@ module.exports = {
       functions: 80,
       lines: 80,
     },
-    './src/components/**/!(ColumnHeaderSelect|FilterHeaderRow|TableToolbar|RowActionsCell|RowActionsError|StatefulTable|StatefulTableDetailWizard|CatalogContent|FileDrop|HeaderMenu|Dashboard|CardRenderer|Attribute|UnitRenderer|ImageHotspots|ImageControls|PageHero|PageTitle|EditPage|AsyncTable|ImageCard|WizardHeader|TableHead|ColumnResize|DateTimePicker|TimeSeriesCard|BarChartCard).jsx': {
+    './src/components/**/!(ColumnHeaderSelect|FilterHeaderRow|TableToolbar|RowActionsCell|RowActionsError|StatefulTable|StatefulTableDetailWizard|CatalogContent|FileDrop|HeaderMenu|Dashboard|CardRenderer|Attribute|UnitRenderer|ImageHotspots|ImageControls|PageHero|PageTitle|EditPage|AsyncTable|ImageCard|WizardHeader|TableHead|ColumnResize|DateTimePicker|TimeSeriesCard|BarChartCard|DashboardGrid).jsx': {
       statements: 80,
       branches: 80,
       functions: 80,
       lines: 80,
     },
+    './src/components/Dashboard/DashboardGrid.jsx': {
```

Comment:
Why did we have to make exceptions here?

Reply:
I found it difficult to write tests with React Testing Library and their strategy (test the way a user would test) in combination with the underlying react-grid-layout library. I was not able to get drag events working in the test environments and exposing additional functions by export required refactoring that made the code worse.

I can a) spend some more time on it to se if I can find a way to proper test it, b) use enzyme, c) leave as is

Reply:
Yeah that makes sense - I think it would be better tested via cypress in a real browser #1203